website/package.json

```
@@ -15,6 +15,8 @@
     "compile": "ts-node scripts/compile.ts && prettier --write src/posts.ts",
     "check-thumbnails": "ts-node scripts/check-thumbnails.ts",
     "check-authors": "ts-node scripts/check-authors.ts",
+    "check-image-frontmatter": "ts-node scripts/check-image-frontmatter.ts",
+    "lint": "npm run check-thumbnails && npm run check-authors && npm run check-image-frontmatter",
```

```suggestion
    "lint": "yarn check-thumbnails && yarn check-authors && yarn check-image-frontmatter",
```

can we use yarn because we're still using yarn?